extensions/cheetah/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Cheetah Changelog
 
+## [1.0.2] - 2022-12-09
+
+### Fixed
+
+- 修复项目打开次数无法更新
+
+### Change
+
+- 新增选择应用 icon
+- 去除列表防抖
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.